CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - the precondition of biceps:R0034_0 so that it is now easier to satisfy.
 - the precondition of biceps:C5 and biceps:R5052 so that it can be satisfied by devices that do not support updating
   MDS descriptors.
+- SDCri version 5.0.0
```

Comment:
This segment begins with
> ### Changed
> - SDCri version 5.0.0-SNAPSHOT

So which is it? :)

Reply:
removed the old entry